modules/desktop/graphics/ewwbar.nix

```diff
@@ -356,48 +362,39 @@ let
           fi
       }
 
-      set_volume() {
-          audio-ctrl set "$1"
-      }
-
-      mute() {
-          audio-ctrl mut
-      }
-
       get() {
-          volume=$(volume_level)
-          muted=$(is_muted)
+          volume=$(pamixer --get-volume | awk '{print $1 + 0.0}')
```

Comment:
Maybe we could use pamixer argument
`       --get-volume-human
              Get the current volume percentage or the string "muted".
`

Reply:
I recall trying this a while ago. Just tried again and remembered that the limitation comes from eww  - it requires this value to be of type f64.
In addition, **--get-volume-human** returns "muted" when the volume is muted, which is also not f64 and causes issues when applying it to the slider in eww :sweat_smile: